packages/@aws-cdk/cx-api/lib/features.ts

```diff
@@ -89,6 +89,12 @@ export const KMS_DEFAULT_KEY_POLICIES = '@aws-cdk/aws-kms:defaultKeyPolicies';
  */
 export const S3_GRANT_WRITE_WITHOUT_ACL = '@aws-cdk/aws-s3:grantWriteWithoutAcl';
 
+/**
```

Comment:
@rix0rrr this doesn't feel quite right to me.  Am I in the right place?

Reply:
I looked around at the code, seems we need a new contextProviderPlugin- the existing SecurityGroupContextProviderPlugin is just for looking up a security group by id and I don't want to overload it. 

Does that work for you?

Reply:
> this doesn't feel quite right to me. Am I in the right place?

I would say it doesn't need to become a feature flag in this way. Feature flags listed here become new, default behavior (on the next project creation, and in the next major version of CDK).

That's actually not what we're trying to achieve, this is just going to be a switchable feature on a per-Security Group basis, whose default can ALSO be controlled globally by a context key.

I would be fine if you declare this constant just in `security-group.ts` (it could even be private, doesn't need to be exported!), and if you describe the presence of this context key, and specifically its motivation and use, in the docstring of the `disableInlineRules` parameter.

Reply:
I like the sound of that, it will be much simpler just to do it in `security-group.ts`.